distage/distage-testkit-core/src/main/scala/izumi/distage/testkit/services/dstest/DistageTestRunner.scala

```diff
@@ -60,77 +60,89 @@ class DistageTestRunner[F[_]: TagK](
     *  By result you'll got [[MemoizationEnvWithPlan]] mapped to tests wit it's plans.
     *  [[MemoizationEnvWithPlan]] represents memoization environment, with shared [[TriSplittedPlan]], [[Injector]], and runtime plan.
     * */
-  def groupTests(distageTests: Seq[DistageTest[F]]): Iterable[(MemoizationEnvWithPlan, Iterable[(DistageTest[F], OrderedPlan)])] = {
+  def groupTests(distageTests: Seq[DistageTest[F]]): Map[MemoizationEnvWithPlan, Seq[(DistageTest[F], OrderedPlan)]] = {
+
+    final case class EnvByproducts(injector: Injector, envLogger: IzLogger)
+
     // here we are grouping our tests by memoization env
     distageTests.groupBy(_.environment.toMemoizationEnv).flatMap {
       case (memoEnv, grouped) =>
-        val memoizedEnvLogger = IzLogger(getTestRunnerLogLevel(memoEnv.verboseTestRunner)).withCustomContext(testRunnerLogger.customContext)
-        grouped
-          .groupBy(_.environment).map {
-            case (env, tests) =>
-              // make a config loader for current env with logger
-              val config = loadConfig(env, memoizedEnvLogger)
-              val runtimeLogger = EarlyLoggers.makeLateLogger(RawAppArgs.empty, memoizedEnvLogger, config, env.logLevel, defaultLogFormatJson = false)(
-                "memoEnv" -> memoEnv.hashCode()
-              )
+        val configLoadLogger = IzLogger(memoEnv.logLevel).withCustomContext("phase" -> "testRunner")
```

Comment:
Hm, but such behaviour you cannot control config logger via debug property.

Reply:
@Caparow Yeah, because it's on by default - you should bump `logLevel` to filter it out. I could add a debug property to _disable_ config logs, but i think it wouldn't be that useful - it's kinda hard to add persistent system properties everywhere, if you're the kind of person who wants zero-log testing

Reply:
Or we can use an enum instead of bool after all - then you could disable config log by setting `Silent`

Reply:
@neko-kai ah, I just thought that we want to make config loader with test runner logger instead of environment one (because making a config is a part of env preparation rather than test run). Anyway, it's not necessary since we already reduced the number of logs for config loader, and an overwhelming number of cases we will have only one config loader message.

Reply:
Ok in that case let's merge as-is